job/common/src/main/java/alluxio/job/migrate/MigrateConfig.java

```diff
@@ -79,6 +85,13 @@ public boolean isOverwrite() {
     return mOverwrite;
   }
 
+  /**
```

Comment:
The javadoc seems slightly confusing. At first glance, I though it meant deleting source file OR source directory. I'm okay leaving as is but I think it could be a little more clear. Maybe something like "whether to delete source inode" or "whether to delete the source path after migration"?